src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personStatus = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <>
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${personStatus}`
+          : 'I am not married'
+        }
+      </p>
```

Comment:
don't forget about code style
```suggestion
      </h2>
      
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
      
      <p className="Person__partner">
          {isMarried
            ? `${partnerName} is my ${personStatus}`
            : 'I am not married'
          }
      </p>
```

Reply:
Done!